code/obj/flock/cage.dm

```diff
@@ -122,72 +186,118 @@
 			var/list/edibles = list()
 			for(var/obj/O in src.contents)
 				edibles += O
-			if(edibles.len >= 1)
+			if(length(edibles) >= 1)
```

Comment:
```suggestion
			if(length(edibles))
```

Reply:
yeah, this is what I get for copypasting old code - will fix